dotcom-rendering/src/server/render.article.apps.tsx

```diff
@@ -125,3 +132,67 @@ window.twttr = (function(d, s, id) {
 		html: renderedPage,
 	};
 };
+
+/**
+ * blocksToHtml is used by the /Blocks endpoint as part of keeping liveblogs live
+ * It takes an array of json blocks and returns the resulting html string
+ *
+ * @returns string (the html)
+ */
+export const renderAppsBlocks = ({
+	blocks,
+	format: FEFormat,
+	host,
+	pageId,
+	webTitle,
+	ajaxUrl,
+	isAdFreeUser,
+	isSensitive,
+	section,
+	switches,
+	keywordIds,
+	abTests = {},
+	edition,
+}: FEBlocksRequest): string => {
+	const format: ArticleFormat = decideFormat(FEFormat);
+
+	const editionId = isEditionId(edition) ? edition : 'UK';
+
+	const config: Config = {
+		renderingTarget: 'Apps',
+		darkModeAvailable: abTests.darkModeWebVariant === 'variant',
```

Comment:
```suggestion
		darkModeAvailable: true,
```

We're in the apps so we don't want to base dark mode support on the web AB test. This is also what this is set to for `renderArticle` higher up in this file:

https://github.com/guardian/dotcom-rendering/blob/f039eee57119da071ff86de989ac7a8251eccd1e/dotcom-rendering/src/server/render.article.apps.tsx#L27